packages/agent/conf/examples/oid4vci_metadata/knb_babylon.json

```diff
@@ -0,0 +1,168 @@
+{
+  "correlationId": "https://ssi.dutchblockchaincoalition.org/issuer",
+  "overwriteExisting": true,
+  "metadata": {
+    "credential_issuer": "https://ssi.dutchblockchaincoalition.org/issuer",
+    "credential_endpoint": "https://ssi.dutchblockchaincoalition.org/issuer/credentials",
+    "display": [
+      {
+        "name": "Babylon Bank",
+        "description": "Babylon Bank Issuer"
+      }
+    ],
+    "credentials_supported": [
+      {
+        "display": [
+          {
+            "name": "Bank account credential",
+            "description": "Bank account credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://${issuer}/assets/images/icons/Babylon_bank_logo.png",
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          },
+          {
+            "locale": "en-US",
+            "name": "Bank account credential",
+            "description": "Bank account credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://dutchblockchaincoalition.org/assets/images/icons/Babylon_bank_logo.png",
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          },
+          {
+            "locale": "nl-NL",
+            "name": "Bankrekeninggegevens",
+            "description": "Bankrekeninggegevens",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://dutchblockchaincoalition.org/assets/images/icons/Babylon_bank_logo.png",
+              "alt_text": "Babylon bank logo"
+            },
+            "background_image": {
+              "url": "https://i.ibb.co/CHqjxrJ/Babylon_bank_bg.png",
+              "alt_text": "Waves in shades of blue"
+            }
+          }
+        ],
+        "order": [
+          "description",
+          "issuerName",
+          "companyName",
+          "bankAccountNumber",
+          "kvkNumber"
+        ],
+        "credentialSubject": {
+          "description": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Description"
+              },
+              {
+                "name": "Description",
+                "locale": "en-US"
+              },
+              {
+                "name": "Beschrijving",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "issuerName": {
```

Comment:
Issuernam is not part of the credential subject! We have an issuer object

Reply:
Fixed